gstreamer/src/enums.rs

```diff
@@ -95,6 +95,13 @@ impl FlowReturn {
         }
     }
 
+    pub fn into_result_value<T, F: Fn() -> T>(self, func: F) -> Result<T, FlowError> {
```

Comment:
This should be an `FnOnce` as that allows you to do much more in the closure. See signature of `Option::unwrap_or_else()` etc.

Reply:
Is it normal to have do `into_result_value(|| ...)` instead of `into_result_value(...)`? I know `unwrap_or_else()` doesn't need the explicit definition of a closure (otoh it's a bit better to have it explicit, maybe).

Reply:
It also needs the closure. That's why there is unwrap_or and unwrap_or_else